python/pyspark/sql/tests/pandas/test_pandas_transform_with_state.py

```diff
@@ -109,6 +109,7 @@ def _test_transform_with_state_in_pandas_basic(
         input_path = tempfile.mkdtemp()
         self._prepare_test_resource1(input_path)
         if not single_batch:
+            time.sleep(10)
```

Comment:
Is this to ensure that created files will have ordering by created timestamp? Is it need to be such a huge, 10 seconds? 

I see now the test will be spending 10s of seconds just for this. We had put some efforts of reducing test execution time and I don't want to defer the effort to the future. Have we tried this by half (5) or even 1-2 seconds?

Reply:
Yeah +1

Reply:
Yeah good point, I've updated to 2s, let's see if this works.